bip-0039/russian.txt

```diff
@@ -0,0 +1,2048 @@
+регион
+реестр
+режим
+резать
+рейтинг
+река
+реклама
+рельс
+ремень
+ремонт
+рента
+реплика
+ресница
+ресторан
+ресурс
+реформа
+рецепт
+речь
+решать
+решение
+решить
+ржавый
+риск
+рисунок
+ритм
+рифма
+робкий
+родитель
+родной
+рожа
+рождение
+роза
+роль
+роман
+ронять
+роса
+рослый
+россия
+рост
+рота
+роща
+рояль
+рубашка
+рубеж
+рубль
+ругать
+руда
+ружье
+руины
+рука
+руль
+ручей
+ручка
+рыба
+рыдать
+рыжий
+рынок
+рысь
+рыхлый
+рыцарь
+рычаг
+рюкзак
+рюмка
+рябой
+сабля
+салон
+самолет
+сани
+сапог
+сарай
+сатира
+сауна
+сахар
+сбоку
+сбор
+сбыт
+свадьба
+сведение
+свежий
+сверху
+свет
+свеча
+свидание
+свинья
+свист
+свобода
+свойство
+связь
+сделка
+сеанс
+север
+сегмент
+сегодня
+седло
+седой
+сезон
+сейф
+сейчас
+секрет
+сектор
+секунда
+село
+семья
+семя
+сено
+сентябрь
+сердце
+середина
+сержант
+серия
+серый
+сестра
+сеть
+сечение
+сжечь
+сзади
+сибирь
+сигарета
+сигнал
+сиденье
+сизый
+сила
+сильный
+символ
+симптом
+синий
+сирота
+система
+ситуация
+сиять
+сказка
+скала
+скамья
+скважина
+сквозь
+скелет
+скидка
+склад
+скорость
+скотина
+скрипка
+скрыть
+скука
+слабый
+слава
+слегка
+след
+слеза
+слияние
+слово
+слой
+слон
+слуга
+служба
+слух
+случай
+слышать
+слюна
+смело
+смена
+смесь
+смех
+смеяться
+смола
+смысл
+смятение
+снаряд
+снег
+снижение
+снизу
+снимок
+снова
+собака
+собор
+собрание
+событие
+совет
+совсем
+создать
+сознание
+сойти
+сокол
+солдат
+солнце
+соль
+сомнение
+соперник
+сопка
+сорок
+сорт
+сосед
+сосна
+состав
+сосуд
+сотня
+соус
+социолог
+союз
+спальня
+спасать
+спать
+спешить
+спина
+список
+спичка
+спор
+способ
+спутник
+сразу
+среда
+срок
+ссылка
+ставка
+стадо
+стакан
+сталкер
+станция
+старик
+статус
+стая
+ствол
+стекло
+стена
+степь
+стиль
+стихи
+стойка
+столица
+стон
+сторона
+стоянка
+страна
+стрела
+строка
+струя
+студия
+стук
+стул
+ступень
+стыд
+суббота
+субъект
+сувенир
+сугроб
+судно
+судья
+суета
+суждение
+сумерки
+сумка
+сумма
+супруг
+сустав
+сутки
+суть
+сухой
+суша
+существо
+сущность
+сфера
+схема
+сцена
+счастье
+счет
+съезд
+съемка
+сырой
+сырье
+сытый
+сыщик
+сюжет
+сюрприз
+табак
+таблица
+тайна
+также
+такси
+талант
+таможня
+танец
+танк
+танцор
+тарелка
+тахта
+таять
+тварь
+твой
+творить
+театр
+тезис
+текст
+телега
+тело
+тема
+темнота
+темп
+тень
+теория
+теперь
+тепло
+терапия
+термин
+терять
+тесный
+тетрадь
+тетя
+техника
+течение
+течь
+теща
+тигр
+тираж
+титул
+тишина
+ткань
+ткнуть
+товар
+тогда
+тоже
+толк
+толпа
+толчок
+тонкий
+тонна
+топливо
+топор
+торговля
+точка
+точно
+тощий
+трава
+традиция
+трактор
+трамвай
+траншея
+трасса
+тревога
+треск
+третий
+трещина
+трибуна
+триста
+трогать
+тройка
+тронуть
+тропа
+тротуар
+труба
+труд
+трюк
+тряпка
+трясти
+туалет
+туго
+туда
+туловище
+туман
+тундра
+тупик
+турист
+турнир
+туфля
+туча
+тыкать
+тысяча
+тьма
+тюльпан
+тяга
+тяжесть
+тянуть
+убедить
+убыток
+уважение
+угол
+угроза
+удав
+удар
+удача
+удивить
+уезд
+ужас
+ужин
+узел
+узкий
+узнать
+узор
+уйма
+указ
+уклон
+укол
+украина
+уксус
+улица
+улыбка
+ум
```

Comment:
too short

Reply:
why length is a problem? most important will one make mistake while entering word list or not.